wpxmlrpc.podspec

```diff
@@ -1,16 +1,15 @@
 Pod::Spec.new do |s|
   s.name         = "wpxmlrpc"
-  s.version      = "0.8"
+  s.version      = "0.8.2"
   s.summary      = "Lightweight XML-RPC library."
   s.homepage     = "https://github.com/wordpress-mobile/wpxmlrpc"
-  s.license      = { :type => 'MIT', :file => 'LICENSE.md' }
-  s.author       = "WordPress"
+  s.license      = { :type => 'GPLv2', :file => 'LICENSE.md' }
+  s.author       = {"WordPress", "Automattic" => "mobile@automattic.com"}
```

Comment:
I think generally that's correct, but in this case this started as a fork of an existing library. While it's possible to change the license from MIT to GPL, a) I don't think we should in this case, b) it should be discussed first